lib/jsdom/living/nodes/Document-impl.js

```diff
@@ -205,6 +205,9 @@ class DocumentImpl extends NodeImpl {
     if (!privateData.options.parsingMode) {
       privateData.options.parsingMode = "xml";
     }
+    if (!privateData.options.encoding) {
+      privateData.options.encoding = "UTF-8";
```

Comment:
Does capitalization make a difference? If not I think `"utf-8"` is the more prevalent style.

Reply:
Indeed, I also prefer `utf-8` but the test expects `UTF-8` for `document.characterSet` and this appears to be what browsers do as well.

The [spec](https://encoding.spec.whatwg.org/#names-and-labels) suggests that new protocols/formats should use the form `utf-8`, though.

Reply:
Ah. The [spec for `characterSet`](https://dom.spec.whatwg.org/#dom-document-characterset) says it should "return context object’s encoding’s [name](https://encoding.spec.whatwg.org/#name)," which for UTF-8 is indeed UTF-8. `characterSet` definitely isn't by any means a new API, so UTF-8 it is.